guides/common/modules/proc_installing-advisor-engine-using-import-and-export.adoc

```diff
@@ -0,0 +1,30 @@
+[id="installing-advisor-engine-using-import-and-export"]
+= Installing {advisorengine} using import and export
+
+You can transfer the container image from a connected system to a disconnected {ProjectServer}. 
+
+.Prerequisites
```

Comment:
User also needs to have the container image downloaded (but not necessarily enabled) on connected system.